function/pkg/operator/operator_test.go

```diff
@@ -236,6 +251,31 @@ func Test_applyObject(t *testing.T) {
 			want1:   client.NewStatusEntryCreated(testObj),
 			wantErr: false,
 		},
+		{
+			name: "updated with labels and annotations",
+			args: args{
+				c: func() client.Client {
+					result := mockclient.NewMockClient(ctrl)
+
+					result.EXPECT().
+						Get(gomock.Any(), gomock.Any(), gomock.Any()).
+						Return(testObj.DeepCopy(), nil).
+						Times(1)
+
+					result.EXPECT().
+						Update(gomock.Any(), gomock.Any(), gomock.Any()).
+						Return(testObjWithLabelsAndAnnotations.DeepCopy(), nil).
```

Comment:
A good option would be to not receive `gomock.Any()` and to not return copied structure. We can add validation of the given argument by replacing this `Any` method with the desired struct

Reply:
Fixed